shared-objects/servers.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const config = {
+  CLIENT: {
+    HOST: process.env.IT_CLIENT_HOST || "localhost",
+    PORT: process.env.IT_CLIENT_PORT || "3100",
+  }
+}
+
+config.CLIENT.URL = `${config.CLIENT.HOST}:${config.CLIENT.PORT}`
```

Comment:
use a function/getter instead https://12factor.net/logs

Reply:
Was hat der Link damit zu tun?
@tofixx Kannst du einfach ein Beispiel geben?

Reply:
Vermeide das mehrfache Setzen von Werten und Abhängigen Variablen. 
Der Wert sollte beim Abrufen generiert werden, etwa...
```suggestion
config.CLIENT.URL = ()=>`${config.CLIENT.HOST}:${config.CLIENT.PORT}`;
```